infra/main.parameters.json

```diff
@@ -23,6 +23,18 @@
     "openAiSkuName": {
       "value": "S0"
     },
+    "computerVisionServiceName": {
+      "value": "${AZURE_COMPUTER_VISION_SERVICE}"
+    },
+    "computerVisionResourceGroupName": {
+      "value": "${AZURE_COMPUTER_VISION_RESOURCE_GROUP}"
+    },
+    "computerVisionResourceGroupLocation": {
+      "value": "${AZURE_COMPUTER_VISION_LOCATION}"
+    },
+    "computerVisionSkuName": {
+      "value": "${AZURE_COMPUTER_VISION_SKU=F0}"
```

Comment:
This has the wrong default value, it should be S1

Reply:
Default value in param updated to S1, comment added in main.bicep.